### ISSUE TITLE
Error While Getting an option value

### ISSUE THREAD

**Issue by manikbajaj** [reduxframework/redux-extensions#283]
Dear Team 

I have updated redux 3.6.0.2 and I am facing a major issue as i not getting any value. I have generated my theme embedded bundle from redux builder. After installing this new theme I am facing some major error. I need options value in functions.php it self. but i am not getting the value

here is my code

if ( ! function_exists( 'mytheme_setup' ) ) :

function mytheme_setup() {

	require get_template_directory() . '/admin/admin-init.php';

	load_theme_textdomain( 'mytheme', get_template_directory() . '/languages' );

	add_theme_support( 'automatic-feed-links' );

	add_theme_support( 'title-tag' );

	add_theme_support( 'post-thumbnails' );

	register_nav_menus( array(
		'primary' => esc_html__( 'Primary Menu', 'mytheme' ),
	) );

	add_theme_support( 'html5', array(
		'search-form',
		'comment-form',
		'comment-list',
		'gallery',
		'caption',
	) );

	add_theme_support( 'post-formats', array(
		'aside',
		'image',
		'video',
		'quote',
		'link',
	) );

}
endif; // mytheme_setup
add_action( 'setup_theme', 'mytheme_setup'  );
#add_action('after_setup_theme', 'mytheme_setup');
#add_action('after_switch_theme', 'mytheme_setup');

global $my_theme; //MY opt_name VARIABLE IN OPTION.INIT.php
echo '<pre>';
print_r($my_theme);
die;

**Comment by kprovance** [comment]
Please review:  https://github.com/reduxframework/redux-framework/blob/master/CONTRIBUTING.md

**Comment by manikbajaj** [comment]
Hi Kevin 
Inorder to give you a better vision of what I am trying to do here are a few details which might just help you to understand the issue in a better way. 

I have updated the Redux Framework to 3.6.0.10. We have created theme options for you to check, this theme option lets you enable and disable WordPress admin bar. As per the Redux Documentation, I have initialised my "$opt_name"  as a global variable, but this doesn't work. 

I have sent you the access to my WordPress Dashboard where I have setup this sample theme. here is the support URL generated from the theme I have created. 

http://support.redux.io/?id=BUPiYKWfwW4r6zZcHyQYZBX170w%2F73GSUthvVcUcpGRbT1gISzmfH6A%2FfbqlLxaawUsX4zLbE9ZozdW787x%2BbA%3D%3D

Look forward to hearing from you. 

Cheers !!
